js-apps/meep-frontend/package-lock.json

```diff
@@ -9953,7 +9958,7 @@
       "requires": {
         "is-in-browser": "^1.1.3",
         "symbol-observable": "^1.1.0",
-        "warning": "^4.0.1"
+        "warning": "^3.0.0"
```

Comment:
The "warning" package must be version ^4.0.1. Update all references from 3.0.0 to 4.0.1

Reply:
done